tools/install.sh

```diff
@@ -1,3 +1,4 @@
+function main {
```

Comment:
And function's under 2 spaces needed.
ref: https://github.com/robbyrussell/oh-my-zsh/wiki/Coding-style-guide#good-12